src/minecraft/kotlin/xyz/wagyourtail/unimined/internal/minecraft/patch/forge/CleanroomMinecraftTransformer.kt

```diff
@@ -128,6 +133,8 @@ open class CleanroomMinecraftTransformer(project: Project, provider: MinecraftPr
         config.properties["mcp_to_srg"] = {
             srgToMCPAsTSRG.absolutePathString()
         }
+        config.systemProperties["fml.dev.extrapath"] =
+            provider.mods.getClasspath().joinToString(File.pathSeparator) { it.absolutePath }
```

Comment:
should set this property using jvmargs as that allows lazy. like
<img width="619" height="257" alt="image" src="https://github.com/user-attachments/assets/683f3d5f-a1a3-45e1-b2ca-a7bef2dd968d" />
because that resolves mods too early...

Reply:
Noted